docusaurus/docs/Android/03-ui/04-message-components/05-message-composer.mdx

```diff
@@ -0,0 +1,290 @@
+# Message Composer
+
+`MessageComposerView` replaces `MessageInputView`, addressing most of its pain-points, including
+extensibility, maintainability, etc. Just like the original view, the revamped component allows
+users to participate in the chat by sending messages and attachments.
+
+![Message Composer](../../assets/message_composer.png)
+
+Let's see how to integrate the new `MessageComposerView` in your UI.
+
+## Usage
+
+To use `MessageComposerView`, include it in your XML layout.
+
+```xml
+<io.getstream.chat.android.ui.message.composer.MessageComposerView
+    android:id="@+id/messageComposerView"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content" />
+```
+
+The recommended way of setting up `MessageComposerView` is by binding it to a `MessageComposerViewModel`. This will make it fully functional by setting up any necessary listeners and data handling.
```

Comment:
is binding it to a `MessageComposerViewModel` instance.
or
is binding it to the `MessageComposerViewModel`.